homeassistant/components/google_mail/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "google_mail",
+  "name": "Google Mail",
+  "config_flow": true,
+  "dependencies": ["application_credentials"],
+  "documentation": "https://www.home-assistant.io/integrations/google_mail/",
+  "requirements": ["google-api-python-client==2.70.0", "oauth2client==4.1.3"],
```

Comment:
The `oauth2client` appears to not be used by this (which is good, since its the older library). Is this using `google-auth-oauthlib`?

Reply:
It's just using the URL strings from `oauth2client`. As to your question, I don't think so?

Reply:
Ah! I missed that. Can this get the url from somewhere else? Home Assistant is only using oauth2client because google calendar needs it for the device auth code exchange flow/library but nothing else should really be using it.

Reply:
done